alisww/blases-over-telnet
# fix historic playback

Lots of helper functionality for dealing with SC (colourising, feed season list, team colours, title match)
Fix historic playback to work with new source data

- Support weather (poorly)
- Handle spillover games better
- Show weather (slightly better)
- Fix fetching when games not started
- Compare team colour against background (hardcoded currently, but user configurable Soon)
- Localise ENV urls and make consistent
- Add helper function for calculating contrast colours
- Colourize team based on contrast with background colour in user_settings
- Extend setter for display to be more responsive and allow for more settings
- Add user setting to disable showing snow (not bound)
- Extend context info about ongoing game to include if a game is a title match
- Handle case where FeedSeasonList does not load
- Fix docker-compose file to work with new source
- Add (tentatively) support for running using VCR as a source
- Switch background colour from light to dark mode
- Fix getting team fallback colour
- Fix historic playback
- Make debug printing work with historic game data
- Only fetch and use current team data when replaying historic games
- SCREAMING
- Add spec for historic playback and improve logging
- Add spec for hash set to test mutation inside a block
- Add spec for historic playback to verify game advancement assumptions
- Fix historic playback
- Fix out of range bug when removing final update for game
- Fix sim not being set on first connect
- Use correct value for sim id
